app/views/play/level/tome/SpellView.coffee

```diff
@@ -694,6 +705,41 @@ module.exports = class SpellView extends CocoView
     return if @destroyed
     Backbone.Mediator.publish 'tome:hide-problem-alert', {}
 
+  saveSpade: (e) =>
```

Comment:
Fat arrow unnecessary, Backbone binds event callbacks to the view automatically.